### ISSUE TITLE
Subtitles of ARD/Das Erste are off by 10 hours

### ISSUE THREAD

**Issue by m4thu** [ytdl-org/youtube-dl#22331]
## Checklist

- [x] I'm reporting a broken site support issue
- [x] I've verified that I'm running youtube-dl version **2019.09.01**
- [x] I've checked that all provided URLs are alive and playable in a browser
- [x] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [x] I've searched the bugtracker for similar bug reports including closed ones
- [x] I've read bugs section in FAQ

## Verbose log

```
 youtube-dl -f best -v --all-subs --skip-download https://www.ardmediathek.de/ard/player/Y3JpZDovL2Rhc2Vyc3RlLmRlL3RhdG9ydC8zYmEwN2VkOC0xMzNjLTQ2MWQtYjZkOS1jZmNmYWZkNTExOGE/falscher-hase 
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['-f', 'best', '-v', '--all-subs', '--skip-download', 'https://www.ardmediathek.de/ard/player/Y3JpZDovL2Rhc2Vyc3RlLmRlL3RhdG9ydC8zYmEwN2VkOC0xMzNjLTQ2MWQtYjZkOS1jZmNmYWZkNTExOGE/falscher-hase']
[debug] Encodings: locale UTF-8, fs utf-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2019.09.01
[debug] Python version 3.7.4 (CPython) - Linux-5.2.11-arch1-1-ARCH-x86_64-with-arch
[debug] exe versions: ffmpeg 4.2.1, ffprobe 4.2.1, rtmpdump 2.4
[debug] Proxy map: {}
[ARDBetaMediathek] falscher-hase: Downloading webpage
[ARDBetaMediathek] Y3JpZDovL2Rhc2Vyc3RlLmRlL3RhdG9ydC8zYmEwN2VkOC0xMzNjLTQ2MWQtYjZkOS1jZmNmYWZkNTExOGE: Downloading m3u8 information
[info] Writing video subtitles to: Falscher Hase-Y3JpZDovL2Rhc2Vyc3RlLmRlL3RhdG9ydC8zYmEwN2VkOC0xMzNjLTQ2MWQtYjZkOS1jZmNmYWZkNTExOGE.de.ttml

```

## Description

The extraction of the subtitles is not working as intended. The subtitles start at 10:00:00.000 while the should start at 00:00:00.000.

I quote out of the .ttml file of Tatort: Falscher Hase:

<tt:p xml:id="C1" region="R1" style="S2" begin="10:00:00.000" end="10:00:02.200">
<tt:span style="S3">UNTERTITEL: Hessischer Rundfunk</tt:span>
</tt:p>
<tt:p xml:id="C2" region="R1" style="S2" begin="10:00:02.200" end="10:00:04.520">
<tt:span style="S3">UNTERTITEL: Hessischer Rundfunk</tt:span>
</tt:p>
<tt:p xml:id="C3" region="R1" style="S2" begin="10:00:06.520" end="10:00:10.000">
<tt:span style="S4">* Tatort-Titelmelodie *</tt:span>
</tt:p>
<tt:p xml:id="C4" region="R1" style="S2" begin="10:00:35.400" end="10:00:38.080">
<tt:span style="S4">* melancholische klassische Musik *</tt:span>

After one hour in the movie has passed, the times counts up to 11:00 instead of 01:00

<tt:p xml:id="C800" region="R1" style="S2" begin="10:59:56.560" end="10:59:58.000">
<tt:span style="S7">Falscher Hase.</tt:span>
</tt:p>
<tt:p xml:id="C801" region="R1" style="S2" begin="11:00:08.160" end="11:00:10.160">
<tt:span style="S7">Der ist ja ganz saftig.</tt:span>
</tt:p>
<tt:p xml:id="C802" region="R1" style="S2" begin="11:00:13.120" end="11:00:16.680">
<tt:span style="S7">Brix, willst du mal probieren?</tt:span>
<tt:br/>
<tt:span style="S7">Der ist sensationell saftig.</tt:span>

**Comment by adrianheine** [comment]
The subtitle downloaded is https://classic.ardmediathek.de/subtitle/330237.

**Comment by m4thu** [comment]
Yes, but where does the time information for the text come from? it is all off by ten hours.

**Comment by basicmaster** [comment]
This is actually an issue on the broadcaster side - the used EBU-TT-D(-Basic-DE) profile with a `timebase` of `media` requires that all timecodes align to the video timeline i.e. start at zero. 

The ten hours are a "magic value": It is common that on production side timecodes start at 10:00:00 (instead of 00:00:00). This has historical reasons; it allows to have a timecode for things before the actual content. But for distribution this shall not be the case, as described. The ARD Mediathek player seems to handle that case i.e. automatically subtracts ten hours in such case.

I will contact the relevant colleagues about this next week...

**Comment by dadosch** [comment]
In case anyone stumbles upon this and wants to embed the subtitles in to the video files: With a future version of MP4Box (software that can embed the subtitles into the mp4 file) it is possible to specify the subtitle zero: https://github.com/gpac/gpac/issues/1618

**Comment by dirkf** [comment]
Continuation (duplicate) of #11562.

Has the timebase been changed at ARD?

We should be able to map 10:00:00 to 0:00:00 if the range of sttl timestamps is in 10:00:00..10:00:00+duration.
